A minor flaw in the code

To anyone,

I have discovered a really nonimportant problem in the code (subtraction is somewhat wrong); the unit test fails on subtract function. I do not know how to fix this. Any help is appreciated.

Mait

--


    def test_subtract():
>       assert subtract(2, 3) == -1
E       assert 5 == -1
E        +  where 5 = subtract(2, 3)

--- a/example.py
+++ b/example.py
@@ -8,7 +8,7 @@ def test_add():
 
 
 def subtract(a, b):
-    return a + b  # <--- fix this in step 8
+    return a - b  # <--- fix this in step 8
 
 
 # uncomment the following test in step 5

